test/e2e/lib/pages/signup/start-page.js

```diff
@@ -25,7 +25,7 @@ export default class StartPage extends AsyncBaseContainer {
 		return await driverHelper.waitTillPresentAndDisplayed( this.driver, By.css( '.step-wrapper' ) );
 	}
 
-	static getStartURL( { culture = 'en', flow = '', query = '' } = {} ) {
+	static getStartURL( { culture = 'en', flow = 'domains', query = '' } = {} ) {
```

Comment:
Do we still need this change? It probably doesn't change anything but I thought I'd ask.

Reply:
Yes, we need it for just one special case at the moment: when an existing user is trying to create another site at https://github.com/Automattic/wp-calypso/blob/trunk/test/e2e/specs/wp-launch-spec.js#L74

We can log an issue to clean up this change once we remove the [redirect logic from signup controller](https://github.com/Automattic/wp-calypso/blob/f95a0b4292148b27724c574dcdf61c3678d237dd/client%2Fsignup%2Fcontroller.js#L129-L143).